docs/src/tutorial/cython_tutorial.rst

```diff
@@ -165,6 +221,72 @@ which we can try out in the interactive interpreter as follows::
 See, it works! And if you're curious about how much work Cython has saved you,
 take a look at the C code generated for this module.
 
+
+It is always good to check where is the Python interaction in the code with the
+``annotate=True`` parameter in ``cythonize()``. Let's see:
+
+.. figure:: htmlreport.png
+
+The function declaration and return use the Python interpreter so it makes
+sense for those lines to be yellow. Same for the list comprehension because
+it involves the creation of a python object. But the line ``if n % i == 0:``, why?
+We can examine the generated C code to understand:
+
+.. figure:: python_division.png
+
+We can see that some checks happen. Because Cython defaults to the
+Python behavior, the language will perform division checks at runtime,
+just like Python does. You can deactivate those checks by using the
+:ref:`compiler directives<compiler-directives>`.
+
+Now let's see if, even if we have division checks, we obtained a boost in speed.
+Let's write the same program, but Python-style::
```

Comment:
Nice idea to compare the two!
In fact, why not add a third measurement that benchmarks the compiled plain Python code?

Reply:
I added the pure python-cythonized version below.